src/registrar/views/portfolios.py

```diff
@@ -492,138 +495,165 @@ def post(self, request, *args, **kwargs):
-    #     else:
-    #         # (NOTE: only create a MemberInvitation if the e-mail sends correctly)
-    #         MemberInvitation.objects.get_or_create(email=email_address, domain=self.object)
-    #     return redirect(self.get_success_url())
-
-    # def form_valid(self, form):
-
-    #     """Add the specified user as a member
-    #     for this portfolio.
-    #     Throws EmailSendingError."""
-    #     requested_email = form.cleaned_data["email"]
-    #     requestor = self.request.user
-    #     # look up a user with that email
-    #     try:
-    #         requested_user = User.objects.get(email=requested_email)
-    #     except User.DoesNotExist:
-    #         # no matching user, go make an invitation
-    #         return self._make_invitation(requested_email, requestor)
-    #     else:
-    #         # if user already exists then just send an email
-    #         try:
-    #             self._send_member_invitation_email(requested_email, requestor, add_success=False)
-    #         except EmailSendingError:
-    #             logger.warn(
-    #                 "Could not send email invitation (EmailSendingError)",
-    #                 self.object,
-    #                 exc_info=True,
-    #             )
-    #             messages.warning(self.request, "Could not send email invitation.")
-    #         except Exception:
-    #             logger.warn(
-    #                 "Could not send email invitation (Other Exception)",
-    #                 self.object,
-    #                 exc_info=True,
-    #             )
-    #             messages.warning(self.request, "Could not send email invitation.")
-
-    #     try:
-    #         UserPortfolioPermission.objects.create(
-    #             user=requested_user,
-    #             portfolio=self.object,
-    #             role=UserDomainRole.Roles.MANAGER,
-    #         )
-    #     except IntegrityError:
-    #         messages.warning(self.request, f"{requested_email} is already a member of this portfolio")
-    #     else:
-    #         messages.success(self.request, f"Added user {requested_email}.")
-    #     return redirect(self.get_success_url())
+    def _send_portfolio_invitation_email(self, email: str, requestor: User, add_success=True):
+        """Performs the sending of the member invitation email
+        email: string- email to send to
+        add_success: bool- default True indicates:
+        adding a success message to the view if the email sending succeeds
+
+        raises EmailSendingError
+        """
+
+        # Set a default email address to send to for staff
+        requestor_email = settings.DEFAULT_FROM_EMAIL
+
+        # Check if the email requestor has a valid email address
+        if not requestor.is_staff and requestor.email is not None and requestor.email.strip() != "":
+            requestor_email = requestor.email
+        elif not requestor.is_staff:
```

Comment:
We should definitely create a follow-up ticket for this because we are still using .is_staff in alot more places than just this. @abroddrick --tagging you for awareness